### PR TITLE
feat: CXAR-918 Initial PR

### PR DESCRIPTION
**Pull request by drcgjung**
## Description

Introduces the most basic self-describing top-level information.

Fixes [Initial Merge](https://jira.catena-x.net/browse/KA-324)

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up to date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] Copyright and license header are present on all affected files

### Additional information

- To request a review, check out [who's involved](https://projects.eclipse.org/projects/automotive.tractusx/who) to see a list of contributors and committers
- Use the [mailing list](https://accounts.eclipse.org/mailing-list/tractusx-dev) if you are unsure, who to contact, or if want to engage in a general discussion
- Check out our guide on [how to contribute](https://eclipse-tractusx.github.io/docs/oss/how-to-contribute)
- Check out our [Release Guidelines](https://eclipse-tractusx.github.io/docs/release)
- Check out the [Eclipse Foundation contribution guidelines](https://www.eclipse.org/projects/handbook/#contributing)

### CODE REVIEW

**Comment by drcgjung** [comment]
@wjost @Siegfriedk I cannot set/propose a reviewer/assignee directly. Could you take this over (it's just the bootstrap markdown - the actual code will follow).

**Comment by SebastianBezold** [comment]
> @pvollmert - I cannot set/propose a reviewer/assignee directly. Could someone of you take this over (it's just the bootstrap markdown - the actual code will follow). Even Janine (although in the list of commiters) is not correctly setup here.

Hi @drcgjung, should we add you to the "Tractus-X contributor" list? This would grant you `triage` permissions. With that you can request reviews

**Comment by drcgjung** [comment]
> > @pvollmert - I cannot set/propose a reviewer/assignee directly. Could someone of you take this over (it's just the bootstrap markdown - the actual code will follow). Even Janine (although in the list of commiters) is not correctly setup here.
> 
> Hi @drcgjung, should we add you to the "Tractus-X contributor" list? This would grant you `triage` permissions. With that you can request reviews

Angelika has already done that, thank you (see the reviewer assignment ;-)

**Comment by SebastianBezold** [comment]
> > > @pvollmert - I cannot set/propose a reviewer/assignee directly. Could someone of you take this over (it's just the bootstrap markdown - the actual code will follow). Even Janine (although in the list of commiters) is not correctly setup here.
> > 
> > 
> > Hi @drcgjung, should we add you to the "Tractus-X contributor" list? This would grant you `triage` permissions. With that you can request reviews
> 
> Angelika has already done that, thank you (see the reviewer assignment ;-)

Ah ok, so is it seems like a similar issue recently described on the mailing list. You have to set your GitHub handle in your [EF profile](https://accounts.eclipse.org/users/drcgjung)

**Review by carslen — CHANGES_REQUESTED** [review]
All of your files containing a license and copyright header are using a invalid license and copyright header, please refer to [TRG 7.02 - License and Copyright header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02). 
Please clarify who is the copyright holder and use appropriate copyright headers. If the Catena-X Association itself is the copyright holder then this copyright line might be valid. 

The `COPYRIGHT.md` needs clarification as well. 

This PR doesn't cover all required legal documentation, please refer to [TRG 7.01 - Legal documentation](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-01) (`NOTICE.md`and `SECURITY.md` files missing). Please follow linked TRG for expected files.

I'm unhappy with the `CHANGELOG.md` file because there are released versions mentioned which do not exists in this repository nor in the fork. I'd have expected either an empty Changelog or no Changelog file at this point.

Having a `DEPENDENCIES` file in this PR without any source code seems weired as well.

In `README.md` you're referencing to catenax-ng GH and Helm repositories and pages (with broken links). This is not valid as Catenax-NG GH organization might be sun downed after consortia phase and you should host all required artifacts for this component/product in Eclipse-Tractusx GH org instead. 

I recommend to schedule a Open-Source onboarding session with @AngelikaWittek to get license/copyright header stuff clarified.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+more.
+
+* https://projects.eclipse.org/projects/automotive.tractusx/developer
+
+The project maintains the following source code repositories
+
+* https://github.com/eclipse/tractusx
+* https://github.com/catena-x
+* https://github.com/catenax-ng
+
```
Comment (carslen) [inline CONTRIBUTING.md:51]: All mentioned repositories either do not exists, are not part of eclipse-tractusx or are GH orgs.

Instead you should list your own repo here (https://github.com/eclipse-tractusx/knowledge-agent).

File: DEPENDENCIES
Comment (carslen) [inline DEPENDENCIES:1]: This PR should not contain the `DEPENDENCIES` file, as there is no source code in this PR. Having the `DEPENDENCIES` file in #2 is totaly fine and PR #2 will be used for the intitial contribution and its IP-Check.

File: README.md [hunk 2]
```diff
@@ -1 +1,87 @@
...
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+-->
+
+# Tractus-X Knowledge Agents Reference Implementations (KA-RI)
+
+![GitHub contributors](https://img.shields.io/github/contributors/eclipse-tractusx/knowledge-agents)
+![GitHub Org's stars](https://img.shields.io/github/stars/catenax-ng)
```
Comment (carslen) [inline README.md:23]: Do not link to catenax-ng repositories/GH Org.

File: README.md [hunk 3]
```diff
@@ -1 +1,87 @@
...
+![GitHub](https://img.shields.io/github/license/eclipse-tractusx/knowledge-agents)
+![GitHub all releases](https://img.shields.io/github/downloads/eclipse-tractusx/knowledge-agents/total)
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=eclipse-tractusx_knowledge-agents&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=eclipse-tractusx_knowledge-agents)
+
+Tractus-X Knowledge Agents Reference Implementations (KA-RI) is a product of the [Catena-X Knowledge Agents Kit (about to move to: Tractus-X Knowledge Agents Kit)](https://catenax-ng.github.io/product-knowledge) implementing the "binding" modules of the CX-0084 standard (Federated Queries in Dataspaces).
+
```
Comment (carslen) [inline README.md:29]: Do not link to catenax-ng repositories/GH Org.

File: README.md [hunk 4]
```diff
@@ -1 +1,87 @@
...
+- Binding Agents are needed by any Agent-Enabled dataspace provider in order to link the Dataspace/Protocol layer to the actual Data and Functional Resources. Binding (sometimes: Bridging) is a technology which rather translates between protocols rather than translating data (Mapping).
+  - [Provisioning Agent (KA-PROV)](provisioning) An agent implementation which binds typical SQL-based backend data sources to SPARQL.
+  - [Remoting Agent (KA-RMT)](remoting) An agent implementation which binds typical REST services to SPARQL.
+- [Conforming Agent (KA-CONF)](conforming) An agent implementation which may play any role (Matchmaking Agent, Binding Agent, EDC Transfer) in order to test the conformity of all other parts of the architecture/standard.
+
+Included in this repository are ready-made [Helm charts](charts). 
+They can be installed from the [Catena-X Knowledge Agents Kit Helm Repository (about to move to: Tractus-X Knowledge Agents Kit Helm Repository)](https://catenax-ng.github.io/product-knowledge/infrastructure).
```
Comment (carslen) [inline README.md:56]: After the initial contribution you should have GH workflows in place which will release your application and other artifacts like Helm chart.

Do not link to catenax-ng repositories/GH Org.

File: README.md [hunk 5]
```diff
@@ -1 +1,87 @@
...
+```shell
+./mvnw -s settings.xml publish
+```
+
+### Deployment
+
+Deployment can be done
+* via [JAR libraries](https://github.com/orgs/eclipse-tractusx/packages?repo_name=knowledge-agents&ecosystem=maven) copied into your Java runtime
+* via [Docker images](https://github.com/orgs/eclipse-tractusx/packages?repo_name=knowledge-agents&ecosystem=docker) 
+* vial [Helm Charts](https://catenax-ng.github.io/product-knowledge/infrastructure/)
```
Comment (carslen) [inline README.md:79]: After the initial contribution you should have GH workflows in place which will release your application and other artifacts like Helm chart.

Do not link to catenax-ng repositories/GH Org.

**Review by carslen — CHANGES_REQUESTED** [review]
Thanks you for addjusting the license/copyright headers! Please refer to my additional comments.

File: CONTRIBUTING.md (hunk 1 above)
Comment (drcgjung) [reply CONTRIBUTING.md:48]: took the https://github.com/eclipse-tractusx/bpdm/blob/main/CONTRIBUTING.md

File: DEPENDENCIES
Comment (drcgjung) [reply DEPENDENCIES:1]: removed

File: README.md (hunk 2 above)
Comment (drcgjung) [reply README.md]: switched to eclipse-tractusx

File: README.md (hunk 4 above)
Comment (drcgjung) [reply README.md]: switched to the eclipse-tractusx helm repo(s)

File: README.md (hunk 3 above)
Comment (drcgjung) [reply README.md:29]: choose a neutral tractusx url alias that can be moved/or replaced when the Kit is released

**Comment by Siegfriedk** [comment]
Merging now as the initial contribution ticket is solved and we will review this again starting next week wi

**Pull request merged**